sdk/identity/identity/review/identity.api.md

```diff
@@ -64,10 +70,15 @@ export class DefaultAzureCredential extends ChainedTokenCredential {
 
 // @public
 export class DeviceCodeCredential implements TokenCredential {
-    constructor(tenantId: string, clientId: string, userPromptCallback: DeviceCodePromptCallback, options?: IdentityClientOptions);
+    constructor(clientId: string, userPromptCallback: DeviceCodePromptCallback, options?: DeviceCodeCredentialOptions);
```

Comment:
Again, same comment on tenantId as in AuthorizationCodeCredential

Reply:
Added in tenantId as a required parameter and added '| organizations' just to document it's existence.

I also added this doc string:
*                 'organizations' may be used when dealing with multi-tenant scenarios.